quiz.js

```diff
@@ -17,6 +17,15 @@ db.dropDatabase();
 
 // your code begins here
 
+db.westeros.insert({houses:{name:"Arryn",
```

Comment:
Mostly right, but the instructions say to call your collection 'houses', not 'westeros'. Plus, since our database is also called 'westeros', we would probably also want to try to avoid confusion by giving our collection a different name.